codegen/src/main/scala/overflowdb/codegen/CodeGen.scala

```diff
@@ -1284,11 +1284,13 @@ class CodeGen(schema: Schema) {
              |  * Traverse to nodes where $nameCamelCase matches one of the elements in `values` exactly.
              |  * */
              |def ${nameCamelCase}Exact(values: $baseType*): Traversal[NodeType] = {
-             |  //fixme: use the index
-             |  val vset = values.to(Set)
-             |  traversal.filter{node => vset.contains(node.${nameCamelCase})}
+             |  if (values.size == 1)
```

Comment:
size == 1 should never happen because we have a method overload for single values.

Reply:
It's not the most common case but I believe it can happen when it's called with an existing sequence, i.e. `.fooExact(someSequence:_*)`

Reply:
Good point.